k2/torch/bin/CMakeLists.txt

```diff
@@ -4,13 +4,15 @@ include(transform)
 set(bin_dep_libs
   ${TORCH_LIBRARIES}
   k2_torch
-  k2_fbank
+  kaldifeat_core
 )
 
+set(feature_srcs ../csrc/features.cc)
+
 #----------------------------------------
 #       CTC decoding
 #----------------------------------------
-set(ctc_decode_srcs ctc_decode.cu)
+set(ctc_decode_srcs ctc_decode.cu ${feature_srcs})
```

Comment:
It will recompile feature_srcs for each binary. Shall we make it a library that can be shared?

Reply:
OK, I thinks so. so, let's change back to its original